bin/database.py

```diff
@@ -1115,6 +1115,20 @@ def upgrade_to_31():
     config.db.sessions.update_many({'subject.firstname_hash': {'$exists': True}}, {'$unset': {'subject.firstname_hash':""}})
     config.db.sessions.update_many({'subject.lastname_hash': {'$exists': True}}, {'$unset': {'subject.lastname_hash':""}})
 
+def upgrade_to_32_closure(coll_item, coll):
+    permissions = coll_item.get('permissions', [])
+    for permission_ in permissions:
+        if permission_.get('site', False):
+            del permission_['site']
+    result = config.db[coll].update_one({'_id': coll_item['_id']}, {'$set': {'permissions' : permissions}})
+    if result.modified_count == 0:
+        return "Failed to remove site field"
+    return True
+
+def upgrade_to_32():
+    for coll in ['acquisitions', 'groups', 'projects', 'sessions']:
+        cursor = config.db[coll].find({'permissions.site': {'$exists': True}})
+        process_cursor(cursor, upgrade_to_32_closure, context = coll)
 
```

Comment:
Currently it should only hold the current site's `id`, `name`, and `site_url` which are all available in the config.